sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -55,11 +55,18 @@
 @ServiceClient(builder = DigitalTwinsClientBuilder.class, isAsync = true)
 public final class DigitalTwinsAsyncClient {
     private static final ClientLogger logger = new ClientLogger(DigitalTwinsAsyncClient.class);
-    private ObjectMapper mapper;
+    private final ObjectMapper mapper;
     private final DigitalTwinsServiceVersion serviceVersion;
     private final AzureDigitalTwinsAPIImpl protocolLayer;
     private static final Boolean includeModelDefinitionOnGet = true;
     private final JsonSerializer serializer;
+    private static final String DIGITAL_TWINS_TRACING_NAMESPACE_VALUE = "Microsoft.DigitalTwins";
```

Comment:
Q- is this something that is assigned to us? any reason it is "Microsoft.DigitalTwins" and not "Azure.DigitalTwins' (similar to our package name having "Azure" in it, and not "Microsoft")?

Reply:
The .NET diagnostics required the resource provider namespace, so I stayed consistent here.

Reply:
@abhipsaMisra The list of resource provider names can be found here - https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-services-resource-providers

Looks like `Microsoft.DigitalTwins` is the right namespace to use.